charts/tfy-gpu-operator/templates/aks-nvidia-device-plugin.yaml

```diff
@@ -56,11 +56,16 @@ spec:
             allowPrivilegeEscalation: false
             capabilities:
               drop: ["ALL"]
+          env:
```

Comment:
I am not sure if the `mcr.microsoft.com/oss/nvidia/k8s-device-plugin:1.11` supports it, I'll test, if it does not work then I'll switch to `nvcr.io/nvidia/k8s-device-plugin:v0.14.1`